sdk/test/Services/DynamoDBv2/UnitTests/Custom/DynamoDBTests.cs

```diff
@@ -328,6 +328,30 @@ public void TestVersionAttributeRename()
             Assert.AreEqual(document["V"].AsInt(), 1);
         }
 
+        [TestMethod]
+        [TestCategory("DynamoDBv2")]
+        public void TestGetTargetTable_DescribeTableDisabled()
+        {
+            var dynamoDBContextConfig = new DynamoDBContextConfig
+            {
+                SkipDescribeTableLoad = true,
+                TableNamePrefix = "DotnetTest-"
+            };
+            var dynamoDBClient = new Mock<IAmazonDynamoDB>();
+            var dynamoDBContext = new DynamoDBContext(dynamoDBClient.Object, dynamoDBContextConfig);
+            var dynamoDBFlatConfig = new DynamoDBFlatConfig(null, dynamoDBContextConfig);
+
+            var itemStorageConfig = dynamoDBContext.StorageConfigCache.GetConfig<Employee>(dynamoDBFlatConfig);
+
+            var table = dynamoDBContext.GetUnconfiguredTable("DotnetTest-EmployeeDetails", skipDescribeTableLoad: true);
+            Assert.IsNotNull(table);
+            Assert.AreEqual("DotnetTest-EmployeeDetails", table.TableName);
+
+            table = dynamoDBContext.GetTargetTable(itemStorageConfig, dynamoDBFlatConfig);
```

Comment:
1. Related to the above comment about friend assemblies, could we just use `dynamoDBContext.GetTargetTable<Employee>();` since that's already public? It's unclear from the test name why we want to test the internal methods, comments may help.
2. Can we assert the full contents of the `Table` object we get out? Don't think we're doing that explicitly anywhere (though the integ tests are making use of it).

Reply:
Done